image.yaml

```diff
@@ -77,15 +78,15 @@ modules:
           - name: os-eap-extensions
           - name: openshift-layer
           - name: openshift-passwd
-          - name: os-logging
+          - name: jboss.container.util.logging.bash
           - name: jboss.container.eap.prometheus.config
             version: '7.2'
           - name: os-eap-txnrecovery.run
             version: 'python3'
           - name: os-eap-python
             version: '3.6'
-packages:
-      content_sets_file: content_sets.yml
+            #packages:
```

Comment:
is this comment on purpose?

Reply:
Nope, thanks for catching, will update in a sec.